### PR TITLE
[55803] Allow free input of status % complete in administration

### PR DESCRIPTION
**Pull request by cbliard**
https://community.openproject.org/wp/55803

In Administration > Work packages > Statuses, when creating or editing a status, the % complete field has been replaced by an input field. Before it was a select with 10% steps which made it impossible to set values like "45%".

### CODE REVIEW

File: app/views/statuses/_form.html.erb [hunk 1]
```diff
@@ -31,12 +31,7 @@ See COPYRIGHT and LICENSE files for more details.
 
 <section class="form--section">
   <div class="form--field -required"><%= f.text_field "name", required: true, container_class: "-middle" %></div>
-  <div class="form--field">
-    <%= f.select :default_done_ratio,
-                 (0..100).step(10).map { |r| ["#{r} %", r] },
-                 container_class: "-xslim",
-                 label: WorkPackage.human_attribute_name(:done_ratio) %>
-  </div>
+  <div class="form--field"><%= f.number_field "default_done_ratio", min: 0, max: 100, container_class: "-tiny" %></div>
```
Comment (dombesz) [inline app/views/statuses/_form.html.erb]: The input field is too small in Firefox MacOS. Due to the stepper arrows, it doesn't fit 2 digit numbers.

image

**Review by dombesz — CHANGES_REQUESTED** [review]

File: app/views/statuses/_form.html.erb (hunk 1 above)
Comment (cbliard) [reply app/views/statuses/_form.html.erb]: good catch, I reduced it to a minimum, but did not check with all browsers. I'll fix that.

Comment (cbliard) [reply app/views/statuses/_form.html.erb]: @dombesz It should be fixed now. Can you test it again please?

**Review by dombesz — APPROVED** [review]
Works as expected, thanks!

**Pull request merged**